osmo-test-5/osmosis.zone_assets.json

```diff
@@ -104,6 +104,12 @@
       "base_denom": "uixo",
       "path": "transfer/channel-1637/uixo",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "archwaytestnet",
+      "base_denom": "aconst",
```

Comment:
This asset is missing an image reference at the Cosmos Chain Registry.
I would suggest indicating this as a testnet_mintage of mainnet ARCH, but due to the different base denoms (aarch vs aconst), I'm not sure if this is the testnet equivalent to ARCH. Please register `images` for const at the Chain Registry.

Reply:
Ok, I will create PR to register images at cosmos Chain Registry

Reply:
I added PR for archway testnet images and logo URIS to Cosmos chain registry here: https://github.com/cosmos/chain-registry/pull/4917